### ISSUE TITLE
Make winner of a/b test on /download/thanks page the default, localize to target langs w. newsletter emails

### ISSUE THREAD

**Issue by ejregithub** [mozilla/bedrock#6072]
From the test /download/thanks page mail CTA content - a/b test #5943

Success Criteria:

- [x] Version A is made the default content and shown to: en. es, pt-br, fr, ru, ID, DE, PL
- [ ] Content is localized to en, es, pt-br, fr, ru, ID, DE, PL corresponding to newsletters available in those langs

**Comment by stephaniehobson** [comment]
I am concerned that the new page does not include the "Try downloading again" prompt that the old one did.

**Comment by ejregithub** [comment]
@stephaniehobson - can you please confirm if I'm seeing the right thing here?  I just went to https://www.mozilla.org/en-US/firefox/download/thanks/?v=a  (and the b version_ and was met with this banner in each:

<img width="1072" alt="screen shot 2018-08-23 at 5 05 53 pm" src="https://user-images.githubusercontent.com/20547886/44558083-0b35ac80-a6f7-11e8-8599-8622e2786cde.png">

**Comment by craigcook** [comment]
There are a few additional updates pending to the page, namely with the after-subscribe thank you message. The desire is to hide the title and tagline and add a Firefox mobile promo below the thanks message. That was the original design but a didn't get implemented due to my oversight. It's not a major change but will take a little JavaScripting to listen for the ajax response.

**Comment by stephaniehobson** [comment]
Okay, I see it now. I just had a bit of banner blindness.

**Comment by alexgibson** [comment]
My 2 pence on this request (after some brief discussion with Amy yesterday):

- If we only localize this new page for the locales requested in the description, we'll have to wait for those translations being completed before we can merge this PR, which would likely delay this work rather than make it happen quicker. We'll also still be in a situation where every other locale is being served the old template, so we're automatically making technical debt for ourselves.
- If instead we chose to localize this page fully, we could merge straight away, because we could use an l10n conditional to serve up the old template e.g.

```python
if lang_file_is_active('firefox/download/thanks', locale):
  template_name = 'firefox/new/scene2-email.html'
else:
  template_name = 'firefox/new/scene2.html'
```

If we did this, locales could opt into the new page independently of any further code changes. We'd need to show some other content for locales that don't get the newsletter however. Some mobile download buttons may be a good choice here.
